common/Resources/ti.internal/extensions/node/buffer.js

```diff
@@ -50,7 +52,34 @@ const floatArray = new Float32Array(1);
 const uint8FloatArray = new Uint8Array(floatArray.buffer);
 
 class Buffer {
-	constructor(tiBuffer, start, length) {
+	/**
+	 * Constructs a new buffer.
+	 *
+	 * Primarily used internally in this module together with `newBuffer` to
+	 * create a new Buffer instance wrapping a Ti.Buffer.
+	 *
+	 * Also supports the deprecated Buffer() constructors which are safe
+	 * to use outside of this module.
+	 *
+	 * @param {integer[]|Buffer|integer|string|Ti.Buffer} arg
```

Comment:
<!--
  0 failure: 
  1 warning:  common/Resources/...
  
  
  DangerID: danger-id-Danger;
  File: common/Resources/ti.internal/extensions/node/buffer.js;
  Line: 64;
-->

- :warning: common/Resources/ti.internal/extensions/node/buffer.js line 64 – Missing JSDoc parameter description for 'arg'. (valid-jsdoc)